.github/workflows/build-executable-test-deploy.yml

```diff
@@ -117,7 +121,7 @@ jobs:
      - name: Build gProfiler executable
        run: |
          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
-         docker buildx create --name multiarch --driver docker-container --use
+         docker buildx create --name multiarch --driver docker-container --use --node multiarch0
```

Comment:
What is `multiarch0`?

Reply:
The default "node name" it uses for the buildx builder with `--name multiarch`